src/contexts/User.tsx

```diff
@@ -1,10 +1,11 @@
 import React, { createContext, useContext, useEffect, useMemo, useReducer } from 'react';
 
-import UserReducer, { UserAction } from '../reducers/User.Reducer';
-import useFirebaseAuth from '../hooks/useFirebaseAuth';
-import useUserCollection from '../hooks/useUserCollection';
-import { FetchedUser, User } from '../types/';
-import { logUser, usersCollection } from '../firebase/firebase';
+import { FetchedUser, User } from '@app/types';
```

Comment:
honestly I was on the fence about a lot of the import plugin fluff but this dependency sorting is very nice, +1